backend/utils/search.py

```diff
@@ -0,0 +1,26 @@
+from mongoengine.queryset.visitor import Q
+from flask import jsonify
+
+
+def model_keywords_search(model, search_fields, search_words_list, search_result=None):
+    try:
```

Comment:
Please use [NumPy style docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html)

Reply:
I've added function description, but I'm not sure about each parameter and return description.
Because in return there is QuerySet type, but I can't set this type for return in docstring.
Could you please advice which type is right for QuerySet return type

Reply:
I think this looks good. Thanks.